src/bin/pg_autoctl/cli_service.c

```diff
@@ -424,28 +424,33 @@ cli_service_status(int argc, char **argv)
 
 	if (!file_exists(pathnames->pid))
 	{
-		log_info("pg_autoctl pid file \"%s\" does not exists", pathnames->pid);
+		log_debug("pg_autoctl pid file \"%s\" does not exists", pathnames->pid);
 
 		if (pg_setup_is_running(pgSetup))
 		{
 			log_fatal("Postgres is running at \"%s\" with pid %d",
```

Comment:
I don't really understand this log message now. It seems like it would be quite confusing to get both this one and the one below.

Reply:
I have added comments about the situation. In maintenance mode, operators can do what they want. Including stop pg_autoctl, restart Postgres, etc.